tests/xcom_test.py

```diff
@@ -1,33 +1,123 @@
 """Test XCOM data queries."""
 
-from __future__ import print_function
 import numpy as np
 from becquerel.tools import xcom
 import pytest
 
-# pylint: disable=protected-access,too-many-public-methods
 XCOM_URL_ORIG = xcom._URL
 
 
 # energies to query using energies keyword
-ENERGIES_3 = [60., 662., 1460.]
+ENERGIES_3 = [60.0, 662.0, 1460.0]
 
 
 # standard grid energies for Germanium from 1 keV to 10 MeV
```

Comment:
this is unfortunate. maybe worth #noqa if that still works with black?

Reply:
Not sure what you mean. Because they aren't the actual line energies anymore with one more sig fig?

Reply:
Oh, I meant to point out the code below "standard grid energies" is now a long list of one entry per line. You do raise a good point though.

Reply:
Oh, I see. I'm not worried about those numbers being the right energies. Yes, the formatting of the lists below are much less compact, but I don't think we can do anything about that.